conanfile.py

```diff
@@ -48,9 +46,16 @@ def source(self):
         extracted_dir = 'x264-snapshot-%s-2245' % self.version
         os.rename(extracted_dir, self._source_subfolder)
 
+    @property
```

Comment:
What is the purpose of this?

Reply:
I don't know. It was commited in https://github.com/bincrafters/conan-libx264/pull/8/commits/363c42a2c450054a790a3f5cee9c532127126642 and seems to use an undocumented feature of conan: https://github.com/conan-io/conan/blob/9e348424d85fa22b41c93cec4b0a113f25fef24d/conans/model/conan_file.py#L159
@akemimadoka can you explain us ?

Reply:
Despite no answer we might go ahead as @SSE4 approved that in #8 once...